packages/salesforcedx-sobjects-faux-generator/package.json

```diff
@@ -10,21 +10,21 @@
     "vscode": "^1.15.0"
   },
   "dependencies": {
-    "@salesforce/salesforcedx-utils-vscode": "40.11.0"
+    "@salesforce/salesforcedx-utils-vscode": "40.11.0",
+    "request-light": "0.2.1",
+    "rxjs": "^5.4.1",
+    "shelljs": "0.7.8",
+    "tree-kill": "^1.1.0"
   },
   "devDependencies": {
     "@types/chai": "^4.0.0",
-    "@types/jsforce": "^1.7.1",
     "@types/shelljs": "0.7.4",
     "@types/mocha": "2.2.38",
     "@types/node": "^6.0.40",
     "@types/rimraf": "^2.0.2",
     "@types/sinon": "^2.3.2",
     "chai": "^4.0.2",
-    "request-light": "0.2.1",
```

Comment:
This is actually used outside of tests so it needs to be moved to the dependencies section.